### PR TITLE
Add disclaimer to imported tags

### PR DESCRIPTION
**Pull request by LeMurphant**
To address https://github.com/StampyAI/stampy-ui/issues/842

Example output metadata:
node devtools.js md 1x9Jn8s4XcLtiP-vVA8crCCg27JdJ3TnAuL8zAMRm4sE
<i>This text was automatically imported from a tag on LessWrong</i>

**Human Values** are the things we care about, and would want an aligned superintelligence to look after and support. It is suspected that true human values are [highly complex](https://www.lesswrong.com/tag/complexity-of-value), and could be extrapolated into a wide variety of forms.

### CODE REVIEW

File: parser/__tests__/parser.test.js [hunk 1]
```diff
@@ -865,7 +865,7 @@ describe("fetchExternalContent", () => {
     fetchMock.mockResponse(JSON.stringify(mockResponse));
     const result = await fetchExternalContent(paragraphs);
     expect(result).toEqual(
-      "This is an example LW tag content (see mockResponse)"
+      {"content": "This is an example LW tag content (see mockResponse)", "sourceName": "LessWrong"}
```
Comment (LeMurphant) [inline parser/__tests__/parser.test.js:868]: @mruwnik I think this makes sense, but I was not certain of the use of the test.

**Comment by LeMurphant** [comment]
Steven was OK with the text. @mruwnik I'd like your OK on the changes I did to the tests

File: parser/parser.js [hunk 2]
```diff
@@ -161,7 +166,17 @@ export const fetchExternalContent = async (paragraphs) => {
   for (const [regex, handler] of tagHandlers) {
     const match = text.match(regex);
     if (match) {
-      return await handler(match.groups.tagName);
+      const content = await handler(match.groups.tagName);
+      return {
```
Comment (mruwnik) [inline parser/parser.js:170]: maybe move the source names into the `tagHandlers` list? So it'd be like:
```
const tagHandlers = [
    [/https:\/\/(www.)?lesswrong.com\/tag\/(?<tagName>[A-z0-9_-]+)/, getLWTag, "LessWrong"],
    (...)
]

for (const [regex, handler, sourceName] of tagHandlers) {
    const match = text.match(regex);
    if (match) {
        return {
           sourceName,
           content: await handler(match.groups.tagName),
       }
   }
}
```

Comment (LeMurphant) [reply parser/parser.js]: Makes sense, how about the current version?

**Review by mruwnik — APPROVED** [review]

**Pull request merged**